packages/console/src/notebook/ScriptEditor.tsx

```diff
@@ -40,9 +44,13 @@ class ScriptEditor extends Component<ScriptEditorProps, ScriptEditorState> {
     isLoaded: false,
     focusOnMount: true,
     onChange: (): void => undefined,
+    onEditorInitialized: (): void => undefined,
+    onEditorWillDestroy: (): void => undefined,
+    updateWorkspaceData: (): void => undefined,
     session: null,
     sessionLanguage: null,
     settings: null,
```

Comment:
Side note, I don't think we need to specific `null` for these params, since we can just leave them as `undefined` (they are both optional).